### DISCUSSION TITLE
nextflow run in docker with conda environment

### DISCUSSION THREAD

**Discussion by COMODr** [nextflow-io/nextflow#2506] [Q&A]
When I run a work flow with nextflow in a docker image, it can excute successefully. But when I run nextflow script with '-with-docker [docker image]', it will get an error "Conda prefix path does not exist or it's not a directory". How can I sove this problem.

**Comment by pditommaso** [comment]
This looks a problem in the configuration. please provide e a replicate test case

**Comment by COMODr** [comment]
Like this attachment example
1、docker run -it test.nf
It can execute successfully
2、nextflow run test.nf -with-docker rnaseq:v1.5.3
conda environment is in the docker image,
the 2d sctript will get this error "Conda prefix path does not exist or it's not a directory"
Thank you very much for your reply and look forward to your help.















At 2021-12-20 17:10:37, "Paolo Di Tommaso" ***@***.***> wrote:

This looks a problem in the configuration. please provide e a replicate test case

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by pditommaso** [reply]
This is a pipeline specific problem, not a NF issue. You should consider reporting it to the corresponding project repo https://github.com/nf-core/rnaseq/

**Comment by COMODr** [comment]
This example is just something I created by myself.  The cause of this problem seems to be the "nextflow run <nfwork> -with-docker rnaseq:v1.5.3" can't map the conda environment path in the script to the Docker container.
Whether it is that conda environment cannot be used in a scripts when executing with Docker containers.


Although the problem was not solved, thank you for your reply in your busy schedule.
















At 2021-12-20 19:38:58, "Paolo Di Tommaso" ***@***.***> wrote:

This is a pipeline specific problem, not a NF issue. You should consider reporting it to the corresponding project repo https://github.com/nf-core/rnaseq/

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
